lib/assert.js

```diff
@@ -416,7 +416,9 @@ function _throws(shouldThrow, block, expected, message) {
       userProvidedMessage &&
       expectedException(actual, expected)) ||
       isUnexpectedException) {
-    fail(actual, expected, 'Got unwanted exception' + message);
+    fail(actual, expected,
+         'Got unwanted exception:\n' +
+         actual.message + message);
```

Comment:
Why is it `'Got unwanted exception:\n' + actual.message + message` instead of `'Got unwanted exception' message + actual.message` (more similar to what it used to be)? This way it would be:

```
Got unwanted exception: explain why it is unwanted.
Message of the unwanted exception.
```